README.md

```diff
@@ -124,19 +157,41 @@ For reference, here is a exp of a step-by-step evaluation:
 6 + 4
 10
 ```
+2 + 3 * 2 + 1
+2 + 6 + 1
+8 + 1
+9
+
 
 ## Problem Seven
 
 Write a step-by-step evaluation for the following expression (remember order of operations): `4 / 2 + 8 / 4`.
 
+4 / 2 + 8 / 4
+2 + 2
+4
+
+
 ## Problem Eight
 
 Write a step-by-step evaluation for the following expression: `'ca' + 'ter' + 'pi' + 'llar'`.
+'cater' + 'pillar'
```

Comment:
should be "cater" + "pi" + "llar" as second step. Always builds from left to right.